src/chainlocks/clsig.cpp

```diff
@@ -6,23 +6,21 @@
 
 #include <tinyformat.h>
 
-namespace llmq {
-    const std::string CLSIG_REQUESTID_PREFIX = "clsig";
+namespace chainlocks {
```

Comment:
consider name `chainlock`.

It's `instantsend` not `instantsends`, it's `wallet` not `wallets`, etc.